### PR TITLE
Added nois testnet

### PR DESCRIPTION
**Pull request by TrevorJTClarke**

### CODE REVIEW

File: testnets/noistestnet/chain.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "cosmos_sdk_version": "0.45",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "cosmwasm_version": "0.30",
+    "cosmwasm_enabled": true,
+    "genesis": {
+      "name": "v1",
+      "genesis_url": "https://raw.githubusercontent.com/noislabs/networks/nois1.final.1/nois-1/genesis.json"
```
Comment (webmaster128) [inline testnets/noistestnet/chain.json:52]:
```suggestion
      "genesis_url": "https://raw.githubusercontent.com/noislabs/networks/nois-testnet-005/nois-testnet-005/genesis.json"
```

**Review by webmaster128 — COMMENTED** [review]
Thank you!

File: testnets/noistestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nois",
```
Comment (JeremyParish69) [inline testnets/noistestnet/chain.json:3]: please change to 'noistestnet'

Comment (TrevorJTClarke) [reply testnets/noistestnet/chain.json]: great catch! I've fixed

**Comment by JeremyParish69** [comment]
> Should we duplicate the logos that are on mainnet or instead reference the mainnet ones from the testnet folder such that we have a single point of truth?

For now, either is fine

**Pull request merged**